src/App.js

```diff
@@ -4,11 +4,6 @@ import CalcApp from './components/Calculator';
 import './components/style.css';
 
 export default class App extends Component {
```

Comment:
[OTIONAL]
- This might have been missed by the previous reviewer but kindly refactor your App.js file to use a functional component.